### ISSUE TITLE
Allow configuring font-display for each font

### ISSUE THREAD

**Issue by YMIndustries** [escaladesports/gatsby-plugin-prefetch-google-fonts#7]
It'd be really nice if this plugin allowed for setting the `font-display` property for each webfont.

There's an article about this property [here](https://developers.google.com/web/updates/2016/02/font-display). It can really improve a page's user experience, especially for users on slower connections. It's also something that Google PageSpeed Insights looks for.

**Comment by JoshuaWalsh** [comment]
I see now that actually font-display IS added, but it's just hardcoded to 'swap'. I still think it would be beneficial for it to be configurable.

**Comment by niicz** [comment]
Yes, please add an option to set font-display individually, swap gives me FOUC.

**Comment by vpicone** [comment]
Google fonts and `google-fonts-plugin` now supports font-display as a parameter. Upgrading to ^4 of google-fonts-plugin would be awesome.
